create_image.py

```diff
@@ -36,3 +66,58 @@
 img_out_path = join(args.output_dir, filename_split[0] + append_name + filename_split[1])
 cv2.imwrite(img_out_path, img_out, [int(cv2.IMWRITE_JPEG_QUALITY), 90])
 
+if args.use_gan:
+    gan_args = argparse.Namespace(
+        approach='adam',
+        lr = .01,
+        beta1 = 0.9,
+        beta2 = 0.999,
+        eps = 1e-8,
+        hmcBeta = 0.2,
+        hmcEps = 0.001,
+        hmcL = 100,
+        hmcAnneal = 1,
+        nIter = 3000,
+        imgSize = IMGSIZE,
+        lam = 0.1,
+        checkpointDir = 'checkpoint',
+        outDir = args.output_dir,
+        outInterval = 200,
+        maskType='custom',
+        centerScale = 0.25,
+        imgs = [img_aligned_path]
+    )
+
+    assert(os.path.exists(gan_args.checkpointDir))
```

Comment:
Don't use asserts in non-unit tests! Use try/catch and log and reraise, for example